src/Serilog/Build/ImplicitDependencyAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Serilog.Build
```

Comment:
I still think `Serilog.Settings.ConfigurationDependencyAttribute` is a more self-describing name and namespace for the type. The attribute is only likely to be used in conjunction with XML or JSON settings, so focusing it on that scenario has some discoverability benefits even if it could be used for other purposes.